adr/0014-home-assistant-supervised.md

```diff
@@ -0,0 +1,80 @@
+# 0014. Installation method: Home Assistant Supervised
+
+Date: 2020-06-03
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This installation method provides the full Home Assistant experience on a regular operating system. This means, all components from the Home Assistant method are used, except for the Home Assistant Operating System. We don’t support using the host system for other applications than the Supervisor.
+
+The Supervisor is not just an application, it is a full appliance that manages the whole system. It will clean up, repair or reset settings to default if they no longer match expected values.
+
+By not using the Home Assistant Operating System, the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. We are not responsible for issues that come up with your host or correcting a wrong configuration.
+
+This method is considered advanced and should only be used if one is an expert in managing a Linux operating system, Docker and networking.
+
+### Supported Operating System, System dependencies and versions
+
+Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. We only support FHS 3.0 on the host file system.
+
+- Docker CE >= 19.03
+- Systemd >= 239
+- NetworkManager >= 1.18.0
+- Avahi >= 0.7
+- AppArmor == 2.13.x
+- Debian Linux Debian 10 aka Buster
+
+Only the above-listed version of Debian Linux is supported for running this installation method. When a new major version of Debian is released, the previous major version is dropped, with a deprecation time of 4 months. An exception to this rule occurs if the new version does not meet the requirements of the Supervisor.
+
+### Additional supported conditions
+
+This installation method can easily be broken if one manages the operating system incorrectly. Therefore, the following additional conditions apply:
+
+- The operating system is dedicated to running Home Assistant Supervised.
+- All system dependencies are installed according to the manual.
+- No additional software, outside of the Home Assistant ecosystem, is installed.
+- Docker needs to be correctly configured to use overlayfs2 storage and journald as the logging driver.
+- NetworkManager is installed but not actively used.
+
+In case any abnormality is detected that prevents Home Assistant from functioning, the Home Assistant Supervisor will report this to the user and block updates to prevent installations from breaking.
+
+### Required Expertise
+
+- **Installation**
+  The user first needs to install a Linux-based operating system (i.e. Debian). They then need to run the installer script.
```

Comment:
```suggestion
  The user first needs to install a Linux-based operating system (i.e. Debian). The user then needs to run the installer script.
```

Reply:
If the only supported version is Debian then shouldn't this read "The user first needs to install Debian 10"